src/lib/types/resources.ts

```diff
@@ -49,7 +49,13 @@ export interface ImageContentItem {
     displayName: string;
 }
 
-type Content = ImageContentItem | VideoContentItem | TiptapContentItem[];
+export type Content = ImageContentItem | VideoContentItem | TiptapContentItem[];
+
+export interface TipTapContent {
+    [key: number]: {
+        tiptap: object;
+    };
+}
```

Comment:
I think this can go away and its usage can be replaced with `TiptapContentItem[]`